content/en/docs/reference/glossary/webhook.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Webhook
+id: webhook
+date: 2019-02-13
+full_link: /docs/reference/access-authn-authz/webhook/
+short_description: >
+  A way for one component to notify another component in real time, via HTTP.
+
+aka:
+tags:
+- networking
+---
+ A notification message sent from one component to another component's unique URL,
+ via HTTP.
+
+<!--more-->
+
+The two components need not be related, and often are not. The sender need only
+know the URL of the recipient side. Typically, a webhook makes a `POST` request
+that includes a payload in JSON format. On the recipient side, the webhook can
```

Comment:
```suggestion
that includes a payload in JSON format. On the recipient side, when it receives the request from the endpoint implementing the webhook, it can
```